### PR TITLE
Add initialisms contained within ACM PCA

### PR DESCRIPTION
**Pull request by RedbackThomson**
Description of changes:
Add initialisms found within the ACM PCA types `CNAME`, `CRL`, `CPS`, `CSR`, `EDI`, `OCSP`, `RFC`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/names/names.go [hunk 1]
```diff
@@ -82,13 +82,19 @@ var (
...
+		{"Cps", "CPS", "cps", nil},
+		{"Csr", "CSR", "csr", nil},
 		{"Dhcp", "DHCP", "dhcp", nil},
 		{"Dns", "DNS", "dns", nil},
 		{"Dpd", "DPD", "dpd", nil},
 		{"Ebs", "EBS", "ebs", nil},
 		{"Ec2", "EC2", "ec2", nil},
 		{"Ecr", "ECR", "ecr", nil},
+		// Prevent "Edit" from becoming "EDIt"
+		{"Edi", "EDI", "edi", regexp.MustCompile("Edi(?!t)", regexp.None)},
```
Comment (jaypipes) [inline pkg/names/names.go:97]: Mind adding a quick unit test that verifies this one?

Comment (RedbackThomson) [reply pkg/names/names.go:97]: Added! And sorted all tests because their ordering was annoying me

File: pkg/names/names_test.go [hunk 2]
```diff
@@ -32,44 +32,46 @@ func TestNames(t *testing.T) {
...
-		{"LdapServerMetadata", "LDAPServerMetadata", "ldapServerMetadata", "ldap_server_metadata"},
+		{"DbiResourceId", "DBIResourceID", "dbiResourceID", "dbi_resource_id"},
 		{"DpdTimeoutAction", "DPDTimeoutAction", "dpdTimeoutAction", "dpd_timeout_action"},
-		{"Iops", "IOPS", "iops", "iops"},
-		{"IoPerformance", "IOPerformance", "ioPerformance", "io_performance"},
-		{"Vlan", "VLAN", "vlan", "vlan"},
+		{"Dynamic", "Dynamic", "dynamic", "dynamic"},
 		{"Ecmp", "ECMP", "ecmp", "ecmp"},
+		{"EdiPartyName", "EDIPartyName", "ediPartyName", "edi_party_name"},
+		{"Editable", "Editable", "editable", "editable"},
```
Comment (RedbackThomson) [inline pkg/names/names_test.go:49]: My new additions

**Comment by RedbackThomson** [comment]
FYI I searched through all `/apis/*.go` files in every service controller directory to find anything that starts with `Edi` and the only thing was `Edit*`

**Review by jaypipes — APPROVED** [review]
:+1: thank you @RedbackThomson!

**Pull request merged**